Removed extraneous back quote

SteveBurkettNZ opened the pull request:
Set-AzureRmVMExtensions code had an unnecessary trailing back quote after the Location parameter.

azurecla commented:
Hi __@mathieu-benoit__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

azurecla commented:
__@mathieu-benoit__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

tysonn commented:
@mathieu-benoit - Nous apprécions la mise à jour du lien.

SteveBurkettNZ committed:
Removed extraneous back quote

Set-AzureRmVMExtensions code had an unnecessary trailing back quote after the Location parameter.
